src/grt/grt-wave_options-parse.ads

```diff
@@ -0,0 +1,64 @@
+--  GHDL Run Time (GRT) - mono-thread version.
+--  Copyright (C) 2005 - 2014 Tristan Gingold
```

Comment:
On 24/07/16 10:21, Jonsba wrote:
> In src/grt/grt-wave_options-parse.ads
> <https://github.com/tgingold/ghdl/pull/121#discussion_r71984179>:
>
>> @@ -0,0 +1,64 @@
>> +--  GHDL Run Time (GRT) - mono-thread version.
>> +--  Copyright (C) 2005 - 2014 Tristan Gingold
>
> Actually since I made some modifications in other packages (like
> Grt.Strings and Grt.Waves), should I also add my name to the copyright
> notice of the corresponding files ?

No, usually only the initial author is marked.
(That's in the GNU coding style that I try to follow).